src/gui/TimeLineWidget.cpp

```diff
@@ -412,11 +413,11 @@ void TimeLineWidget::mouseMoveEvent( QMouseEvent* event )
 				// marking instead of pushing it.
 				if( m_action == MoveLoopBegin ) 
 				{
-					m_loopPos[0] -= TimePos::ticksPerBar();
+					m_loopPos[0] -= m_snapSize * TimePos::ticksPerBar();
```

Comment:
I tried my best to get negative values here (e.g. by changing bar sizes), but without success :+1: Likely because you call `t.quantize()` right before.